tests/test_api/test_endpoints.py

```diff
@@ -295,7 +295,6 @@ def test_submit_workflow_activities(api_site_client):
                     "nmdc:dobj-11-bds7qq03",
                 ],
                 "type": "nmdc:ReadQcAnalysisActivity",
```

Comment:
I'm wondering whether this `type` value has become `nmdc:ReadQcAnalysis` (i.e. no `Activity`) in the Berkeley schema.

Reply:
Tagging @aclum about this for future reference. I'm about to merge this PR, which I think will take this thread further "out of sight, out of mind."